src/app/views/query-runner/query-input/auto-complete/auto-complete.util.ts

```diff
@@ -13,60 +14,38 @@ export function cleanUpSelectedSuggestion(compare: string, userInput: string, se
   return finalSelectedSuggestion;
 }
 
-export function getParametersWithVerb(properties: IAutoCompleteProps) {
-  const { autoCompleteOptions, sampleQuery: { selectedVerb } } = properties;
-  if (!autoCompleteOptions) {
-    return;
-  }
-  const parameters = autoCompleteOptions.parameters;
-  if (!parameters) {
-    return;
-  }
-  return parameters.find(parameter => parameter.verb === selectedVerb.toLowerCase());
-}
-
-export function getLastCharacterOf(content: string) {
+function getLastCharacterOf(content: string) {
   return content.slice(-1);
 }
 
-export function getLastSymbolInUrl(url: string) {
-  const availableSymbols = [
-    {
-      key: '/',
-      value: 0
-    },
-    {
-      key: ',',
-      value: 0
-    },
-    {
-      key: '$',
-      value: 0
-    },
-    {
-      key: '=',
-      value: 0
-    },
-    {
-      key: '&',
-      value: 0
-    },
-    {
-      key: '?',
-      value: 0
-    }
-  ];
-
-  availableSymbols.forEach(element => {
-    element.value = url.lastIndexOf(element.key);
-  });
-  const max = availableSymbols.reduce((prev, current) => (prev.value > current.value) ? prev : current);
-  return max;
-}
-
 // Filter out suggestions that don't contain the user's input
-export function getFilteredSuggestions(compareString: string, suggestions: string[]) {
+function getFilteredSuggestions(compareString: string, suggestions: string[]) {
   return suggestions.filter((suggestion: string) => {
     return suggestion.toLowerCase().indexOf(compareString.toLowerCase()) > -1;
   });
```

Comment:
Suggestion: We can sort the suggestions so that the ones that match the search text appear first on the list as Sunday suggested. We can achieve this with a custom sort that checks how close a suggestion is to the searchtext.
This is not within the scope of this task, but it's something we can implement later